cortex/blender/__init__.py

```diff
@@ -2,7 +2,13 @@
 import re
 import shlex
 import shutil
-import mda_xdrlib as xdrlib
+try:
+    import xdrlib
```

Comment:
This first try/except statement is not necessary. `mda_xdrlib` is in the requirements, so we can just assume it is installed.